torch/_torch_docs.py

```diff
@@ -8292,25 +8292,30 @@ def merge_dicts(*dicts):
 svd(input, some=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
 
 Computes the singular value decomposition of either a matrix or batch of
-matrices :attr:`input`. The singular value decomposition is represented as a
-namedtuple `(U, S, V)`, such that :attr:`input` `= U diag(S) Vᴴ`.
+matrices :attr:`input`.
+
+The singular value decomposition is represented as a
+namedtuple `(U, S, V)`, such that `input = U diag(S) Vᴴ`.
```

Comment:
I'm curious what your thoughts are on https://github.com/pytorch/pytorch/issues/54878#issuecomment-821962806, internally we're hoping to get rid of unicode like Vᴴ because we're concerned about supporting it on multiple platforms.

Reply:
I think LaTeX is definitely the way to go. These `torch.svd` docs are not as polished as  `torch.linalg.svd` (I should have updated them to look the same).
In the end, I just used unicode in some places to write inline maths. The unicode characters that I used are the following:
`≤ ≥ σ₁ σᵢ σₙ S⁻¹ · ε`.

I have to say that inline maths looks nicer in unicode, but I used it so little that it'd be no problem to change it by code or LaTeX. I'll update the PR not to use unicode.